### ISSUE TITLE
Wallet Connect - Renting fails with a RPC URL not available error

### ISSUE THREAD

**Issue by veracomm** [veradefi/vera.financial#22]
**Error:** 
No RPC Url available for chainId:

**Steps:**
1. Connect to BSC chain using Wallet Connnect
2. Go to Rent page and rent a NFT (I used this - https://rent.vera.financial/rent-details/61cc7d55dd2a6b2fbeb52300)
3. Fill rental details - BUSD
4. Click Approve Payment Options - failed

**Error log:**

ObjectbuildDate: "Sat Jan 22 2022"buildMajor: 0buildMinor: 0buildRevision: 2buildTag: "PROD"[[Prototype]]: Object
2.c360804d.chunk.js:2 wallet connect
2.c360804d.chunk.js:2 wc:de1a4e5a-ace5-465b-8870-46277443e9eb@1?bridge=https%3A%2F%2Fu.bridge.walletconnect.org&key=0799fda1682bc2944c89f52e29bafb4a2c5f1bad68ecd175c56ca0b47e08fc28
2.c360804d.chunk.js:2 Uncaught (in promise) Error: No RPC Url available for chainId: 56
    at r.value (2.c360804d.chunk.js:2:2362485)
    at r.<anonymous> (2.c360804d.chunk.js:2:2362110)
    at u (2.c360804d.chunk.js:2:1205695)
    at Generator._invoke (2.c360804d.chunk.js:2:1205483)
    at Generator.next (2.c360804d.chunk.js:2:1206124)
    at n (2.c360804d.chunk.js:2:1781)
    at c (2.c360804d.chunk.js:2:1984)
    at 2.c360804d.chunk.js:2:2043
    at new Promise (<anonymous>)
    at r.<anonymous> (2.c360804d.chunk.js:2:1924)
main.c2571e7f.chunk.js:1 error Error: Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.
    at f.decodeParametersWith (2.c360804d.chunk.js:2:1347503)
    at f.decodeParameters (2.c360804d.chunk.js:2:1347378)
    at T.l._decodeMethodReturn (2.c360804d.chunk.js:2:1337236)
    at h.outputFormatter (2.c360804d.chunk.js:2:1342508)
    at h.formatOutput (2.c360804d.chunk.js:2:423756)
    at o (2.c360804d.chunk.js:2:431604)
    at o (2.c360804d.chunk.js:2:325456)
    at b.run (2.c360804d.chunk.js:2:76630)
    at h (2.c360804d.chunk.js:2:76124)
main.c2571e7f.chunk.js:1 Error: This contract object doesn't have address set yet, please set an address first.
    at Object.ContractNoAddressDefinedError (2.c360804d.chunk.js:2:2592332)
    at Object.l._processExecuteArguments (2.c360804d.chunk.js:2:1341178)
    at Object.l._executeMethod (2.c360804d.chunk.js:2:1341528)
    at main.c2571e7f.chunk.js:1:87336
    at u (2.c360804d.chunk.js:2:1205695)
    at Generator._invoke (2.c360804d.chunk.js:2:1205483)
    at Generator.next (2.c360804d.chunk.js:2:1206124)
    at n (2.c360804d.chunk.js:2:1781)
    at c (2.c360804d.chunk.js:2:1984)

**Comment by veracomm** [comment]
Hi @arbach  and @DLekocaj888 , any updates on this issue?

**Comment by veracomm** [comment]
@arbach as we discussed in the last QA call, can you check if you can repro this issue at your end?

**Comment by DLekocaj888** [comment]
@veracomm Just tried and I successfully rented a NFT without coming across to the above reproduced issue.

**Comment by veracomm** [comment]
What wallet did you use? Metamask or trust wallet for wallet connect?

**Comment by DLekocaj888** [comment]
I used Matamask for wallet connect.

**Comment by veracomm** [comment]
> I used Metamask for wallet connect.

I used Trust Wallet, may be that could be the use. Let me try again in a few hours. @DLekocaj888

**Comment by DLekocaj888** [comment]
@veracomm were you able to face the same issue with the trust wallet?

**Comment by veracomm** [comment]
> @veracomm were you able to face the same issue with the trust wallet?

@DLekocaj888  it's still an issue. I get the same error. (Always been using wallet connect with Trust Wallet)
